_episodes/13-conditionals.md

```diff
@@ -372,6 +372,18 @@ final velocity: 30.0
 > > print(smallest, largest)
 > > ~~~
 > > {: .language-python}
+> > It can be argued that an advantage of using this method would be explicitly checking for each value in the list.
```

Comment:
On further scrutiny I don't find this explanation very satisfying. The efficiency gains are likely minimal, I think the real gain is in clarity and we should emphasize that instead.

Efficiency wise, the second solution is running two passes over the list of `values`, as  `min(values)` and `max(values)` are both iterate over the entire `values` list. 

The most "efficient" method would be to iterate over `values` once, and replace the `min` and `max` calls with simpler conditional checks e.g.,

```
smallest = v if v < smallest
largest = v if v > largest
```

anyways I think we should stick with the clarity argument. Thoughts?

Reply:
Yeah, I agree. I will update that.